Samples/WindowsCopilotRuntime/cs-wpf-dotnet/runDotNetBuild.ps1

```diff
@@ -79,7 +99,7 @@ if (-not (Test-Path $pfx))
 }
 
 Write-Host "Running 'dotnet build /p:platform=$Platform /p:configuration=$Configuration'"
-dotnet build /p:platform=$Platform /p:configuration=$Configuration
+dotnet build /p:restore /p:platform=$Platform /p:configuration=$Configuration 
 
 Write-Host "`nPlease Install '$PSScriptRoot\WCRforWPF\.user\wcrforwpf.certificate.sample.cer' to Local Machine/Trusted People Store before running"
```

Comment:
Why not just do it, or prompt to do it?

Reply:
Would this mean we need an admin prompt to do this?

Reply:
One would be needed, but if that happens just during the "yes, please install this cert" stage, that would be great.